docs/01.CONFIGURATION.md

```diff
@@ -255,6 +255,17 @@ This option is disabled by default.
 | CMake:  | `-DJERRY_SYSTEM_ALLOCATOR=ON/OFF`            |
 | Python: | `--system-allocator=ON/OFF`                  |
 
+### Test262
+
+This option enables the use of $262 object.
+This option is disabled by default.
+
+| Options |                                              |
+|---------|----------------------------------------------|
+| C:      | `-JERRY_TEST262=0/1`               |
```

Comment:
* The closing `|` should be aligned in all lines.
* Missing a `D` prefix: `-DJERRY_TEST262=...`